azure/azure_build.yaml

```diff
@@ -35,14 +35,14 @@ jobs:
         CONFIG_FILE: ${{ parameters.GCC_CONFIG }}
   variables:
   - group: Docker
-  - name: MDOLAB_USER
+  - name: MDOLAB_HOMEDIR
     value: /home/mdolabuser/
   - name: DOCKER_WORKING_DIR
-    value: ${{ variables.MDOLAB_USER }}/repos/${{ parameters.REPO_NAME }}
+    value: ${{ variables.MDOLAB_HOMEDIR }}/repos/${{ parameters.REPO_NAME }}
   - name: DOCKER_MOUNT_DIR
-    value: ${{ variables.MDOLAB_USER }}/azure/${{ parameters.REPO_NAME }}
+    value: ${{ variables.MDOLAB_HOMEDIR }}/azure/${{ parameters.REPO_NAME }}
   - name: BASHRC
-    value: \$HOME/.bashrc_mdolab
+    value: ${{ variables.MDOLAB_HOMEDIR }}.bashrc_mdolab
```

Comment:
Missing the slash

Reply:
Ugh dammit. Thank you for being thorough.

Reply:
Oh I see, you the variable has a slash

Reply:
Yeah I should probably remove that slash right?

Reply:
I've actually been wanting to fix that. There are duplicate slashes in most cases.

Reply:
I'm a few seconds behind. Yes, I agree.